maker-frontend/src/components/Hooks.tsx

```diff
@@ -5,6 +5,7 @@ export default function useLatestEvent<T,>(
     source: EventSource,
     event_name: string,
     mapping: (key: string, value: any) => any = (key, value) => value,
+    filter?: (event: Event) => boolean,
```

Comment:
I haven't found a way to filter events on the event source - happy to change if you know how. Regarding the alternative, I am not so sure about it. I can think of argument for and against it. I guess my key concern here is that this requires our backend to make assumptions about how events are consumed by the client. It might bring benefits for our case, but it might be cumbersome for others. Our events are currently simple changed events - adding what has changed into the event could get very complex very quickly (because where do you draw the line?) Instead I'd suggest we introduce server side filtering using e.g. a query parameter that could also resolve the need for client side filtering.